ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx

```diff
@@ -0,0 +1,351 @@
+    },
+    searchCloseButton: {
+      marginRight: '10px'
+    },
+    searchCloseIcon: {
+      fontSize: '12px !important'
+    }
+  })
+);
+
+export default function PathNavigatorTreeItem(props: PathNavigatorTreeItemProps) {
+  const {
+    node,
+    itemsByPath,
+    keywordByPath,
+    totalByPath,
+    childrenByParentPath,
+    onLabelClick,
+    onIconClick,
+    onOpenItemMenu,
+    onFilterChange,
+    onMoreClick
+  } = props;
+  const classes = useStyles();
+  const [over, setOver] = useState(false);
+  const [showFilter, setShowFilter] = useState(Boolean(keywordByPath[node.id]));
+  const [keyword, setKeyword] = useState(keywordByPath[node.id] ?? '');
+  const { formatMessage } = useIntl();
+  const onMouseOver = (e) => {
+    e.stopPropagation();
+    setOver(true);
+  };
+  const onMouseLeave = (e) => {
+    e.stopPropagation();
+    setOver(false);
+  };
+
+  const onFilterButtonClick = () => {
+    setShowFilter(!showFilter);
+  };
+
+  const onCloseFilterBox = () => {
+    if (keyword) {
+      setKeyword('');
+      onFilterChange('', node.id);
+    }
+  };
+
+  switch (node.id) {
+    case 'loading': {
+      return (
+        <div className={classes.loading}>
+          <CircularProgress size={14} />
+          <Typography variant="caption" color="textSecondary">
+            <FormattedMessage id="words.loading" defaultMessage="Loading" />
+          </Typography>
+        </div>
+      );
+    }
+    case 'empty': {
+      return (
+        <section className={classes.empty}>
+          <ErrorOutlineRoundedIcon />
+          <Typography variant="caption">
+            <FormattedMessage id="filter.noResults" defaultMessage="No results match your query" />
+          </Typography>
+        </section>
+      );
+    }
+    case 'more': {
+      return (
+        <section className={classes.more}>
+          <Button
+            color="primary"
+            size="small"
+            onClick={() => {
+              onMoreClick(node.parentPath);
+            }}
+          >
+            <FormattedMessage
+              id="filter.more"
+              defaultMessage="{count, plural, one {...{count} more item} other {...{count} more items}}"
+              values={{ count: totalByPath[node.parentPath] - childrenByParentPath[node.parentPath].length }}
+            />
+          </Button>
+        </section>
+      );
+    }
+    default: {
+      return (
+        <TreeItem
+          key={node.id}
+          nodeId={node.id}
+          onLabelClick={(event) => onLabelClick(event, node.id)}
+          onIconClick={() => onIconClick(node.id)}
+          label={
+            <>
+              <section className={classes.itemDisplaySection} onMouseOver={onMouseOver} onMouseLeave={onMouseLeave}>
+                <ItemDisplay
+                  styles={{ root: { flexGrow: 1, height: '26px' } }}
+                  item={itemsByPath[node.id]}
+                  showPublishingTarget={true}
+                  showWorkflowState={true}
```

Comment:
These are optional, you may also remove them all together.
```suggestion
                  showPublishingTarget
                  showWorkflowState
```